map.js

```diff
@@ -203,6 +214,27 @@ const app = new Vue({
             initiallyChecked: true,
           });
         }
+
+        if (marchonpollsFlagshipEvents.features.length) {
+          _this.map.addSource('marchopolls-flaghsip-events-geojson', { type: 'geojson', data: marchonpollsFlagshipEvents });
+          _this.addLayer('marchonpolls-flagship-events', 'marchopolls-flaghsip-events-geojson', { 'icon-image': 'star-pink' });
+          _this.mapLayers.push({
+            layerId: 'marchonpolls-flagship-events',
+            label: 'MarchOn Polls Flagship Events)',
```

Comment:
Looks like a typo with the `)` at the end of the label
```suggestion
            label: 'MarchOn Polls Flagship Events',
```